object/object.go

```diff
@@ -153,6 +156,9 @@ func (o *Object) SetSignature(v *neofscrypto.Signature) {
 // Payload returns payload bytes.
 //
 // See also [Object.SetPayload].
+//
+// The value returned shares memory with the structure itself, so changing it can lead to data corruption.
+// Make a copy if you need to change it.
```

Comment:
Usually "see also" is the last thing in the doc, so this hunk better be placed above it (like in other places).

Reply:
Moved